host/bcm283x/bcm283xsmoketest/bcm283xsmoketest.go

```diff
@@ -47,17 +47,24 @@ func (s *SmokeTest) Run(f *flag.FlagSet, args []string) error {
 	}
 
 	start := time.Now()
-	pClk := &loggingPin{bcm283x.GPIO6, start}
+	pDMA := &loggingPin{bcm283x.GPIO6, start}
```

Comment:
keep it as pClk since even if you are doing DMA with it, it's special functionality is to be a GPCLK.

Reply:
done